### ISSUE TITLE
infinite ratio

### ISSUE THREAD

**Issue by mighelch** [BiglySoftware/BiglyBT#1154]
Under which circumstances ration is infinite? I readded previously downloaded torrents, on some it shows ratio on others it's infinite. I mean why some rehashed/readded torrents has "downloaded" value others has 0.

**Comment by ferdnyc** [comment]
@mighelch This is more a guess than an answer, but:

A _new_ torrent will normally show infinite ratio until the first time it's started. So I imagine that readded torrents would probably show their old ratio if they had previously uploaded data, but if they _never_ uploaded any data (meaning, they were removed with ratio `0.000`) then on readd the ratio would again show as ∞.

Hmm... I just tried to test my theory, by removing both a torrent that had `0.000` ratio and a torrent that had a ratio over `1.000`, then readding them. For me, they _both_ came back with initial ∞ ratio when initially added (I added them stopped), but then **both** reset to `0.000` when I started them. So, not very conclusive. But, that may be partly down to how the torrent with the previous ratio was added (using a magnet link from RSSFeed Scanner, rather than a torrent file.)
